### ISSUE TITLE
Amazon Echo integration

### ISSUE THREAD

**Issue by pgrigoruta** [magento-hackathon/developer-paradise-2016#5]

**Comment by pgrigoruta** [comment]
Continue Alan Kent's idea of integrating Magento 2 with Amazon Echo - https://alankent.me/2016/04/18/hey-magento-what-is-my-stock-level/

**Comment by peterjaap** [comment]
Sounds good, are you bringing an Echo?

**Comment by nuzil** [comment]
Nice idea :)

**Comment by pgrigoruta** [comment]
Nope, I don't have one, maybe someone does though. Or we can just integrate with other voice recognition software, just wanted to control Magento 2 verbally :)
